test/parse/module/bad-table-invalid-function.txt

```diff
@@ -0,0 +1,10 @@
+;;; ERROR: 1
+(module
+  (table 1 anyfunc)
+  (func $f)
+  (elem (i32.const 0) $f 1))
+(;; STDERR ;;;
+parse/module/bad-table-invalid-function.txt:5:26: function variable out of range (max 1)
+  (elem (i32.const 0) $f 1))
```

Comment:
Seem a little but odd that the error message is displaying the max in 1-based index space but the error its pointing to is 0-base.   Seems to me that 1 is a valid index if max=1.